dashboard/pages/screening/callbacks.py

```diff
@@ -138,6 +139,11 @@ def on_outlier_purge_stage_entry(
     compounds_count = plates_count * bmg_vals.shape[2] * (bmg_vals.shape[3] - 2)
     outliers_count = (bmg_vals[:, 1] == 1).sum()
 
+    report_data = {}
```

Comment:
How about saving this data into pq/npz and read it during generating a report?

Reply:
I adviced @AndrzejKaj to use the Store as there are not many stats to aggregate (merely a couple of values), thus to have a separate file for storing these is quite suboptimal